lib/database/search.php

```diff
@@ -44,13 +44,14 @@ function performSearch($searchQuery, $offset, $count, &$searchResultsOut)
     (
         SELECT 'Comment' AS Type, cua.User AS ID,
 
-        IF( c.articletype=1, CONCAT( '/Game/', c.ArticleID ),
-            IF( c.articletype=2, CONCAT( '/Achievement/', c.ArticleID ),
-                IF( c.articletype=3, CONCAT( '/User/', ua.User ),
-                    IF( c.articletype=5, CONCAT( '/feed.php?a=', c.ArticleID ), c.articletype )
-                )
-            )
+        CASE
+            WHEN c.articletype=1 THEN CONCAT( '/Game/', c.ArticleID )
+            WHEN c.articletype=2 THEN CONCAT( '/Achievement/', c.ArticleID )
+            WHEN c.articletype=3 THEN CONCAT( '/User/', ua.User ),
```

Comment:
The trailing comma here seems to be the culprit for search not returning any results anymore

Reply:
Good catch, thank!!